content_scripts/tv.js

```diff
@@ -458,15 +458,15 @@ tv.parseReportTable = async () => {
   const strategyHeaders = []
   await page.waitForSelector(SEL.strategyReportHeader, 2500)
   let allHeadersEl = document.querySelectorAll(SEL.strategyReportHeader)
-  if (!allHeadersEl || allHeadersEl.length !== 4)
+  if (!allHeadersEl)
     if (!tv.isParsed)
       throw new Error('Tradingview UI changed. Can\'t get performance headers. Please contact support')
     else
       return {}
-  for(let headerEl of allHeadersEl) {
-    if(headerEl)
-      strategyHeaders.push(headerEl.innerText)
+  for (let i = 0; i < allHeadersEl.length - 1; i++) {
```

Comment:
Needs better handing of new UI extra header. Current solution works only in fullscreen